sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -687,6 +688,20 @@ func (m *marshaller) expandCrossLanguage(namedEdge NamedEdge) (string, error) {
 		EnvironmentId: m.addDefaultEnv(),
 	}
 
+	// Add the coders for output in the marshaller even if expanded is nil
+	// for output coder field in expansion request.
+	// We need this specifically for Python External Transforms.
+	names := strings.Split(spec.Urn, ":")
+	if len(names) > 2 && names[2] == "python" {
```

Comment:
Do we need this section locked to python now that we have the explicit "SetOuputCoder" hack below? I'd rather not lock it to python if a java transform ends up needing it.

Reply:
done.